custom/icds_reports/utils/__init__.py

```diff
@@ -426,15 +426,17 @@ def get_location_filter(location_id, domain, include_object=False):
         sql_location = SQLLocation.objects.get(location_id=location_id, domain=domain)
     except SQLLocation.DoesNotExist:
         return {'aggregation_level': 1}
-    if include_object:
```

Comment:
this is currently used only on fact sheet and was making the fact sheet to query on the one level below.

Reply:
hey, yeah,  this snippet being here causing the aggregation_level to be +2 then what is necessary. If you see line 435 in same function `config['aggregation_level'] = len(config) + 1`. So I have moved the code snippet below this line so that agg_level calculation based on the length of the config does not give +2 level.

+2 because :
1. +1 because of the how it has been happening through the dashboard
2. another +1 because of this code snippet being here

Reply:
ugh, that's nasty. it might be nicer to not make the aggregation level dependent on the length of the config (which is confusing and easy to miss as i apparently did) but instead on the length of the location query below, so that if future changes are made we dont run into this again.